application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -142,4 +146,35 @@ static ReminderRoute toPrivate(PrivateChannel channel) {
                             + " the original channel you wanted it to be send to)");
         }
     }
-}
+
+    private void reminderFailureAction(int id, long channelId, long authorId,
+                                       CharSequence content, long guildId, int reminderFailureCounter) {
+        if (reminderFailureCounter <= 3) {
```

Comment:
prefer to not do:
```java
if (...) {
  some long code
} else {
  sth short
}
```
instead, use early-return:
```java
if (!...) {
  sth short
  return
}

sth long
```
that way, u have no nesting for the long part and its more readable

Reply:
done